style.css

```diff
@@ -35,5 +96,142 @@ img {
     .img-grid img {
         height: 300px;
     }
+}
+
+
+/* About Page */
+.about {
+    font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+    width: 100%;
+    padding: .5rem;
+    color: #36454f;
+    position: relative;
+}
+
+.about-title {
+    width: 100%;
+    margin: 1.5rem;
+    text-align: center;
+    font-size: 3rem;
+}
+
+.about-scrimba {
+    max-width: 70ch;
+    padding: 0 3rem;
+    margin: 1rem auto;
+    font-size: 1.25rem;
+    line-height: 2rem;
+    letter-spacing: .1rem;
+}
+
+.current-dev{
+    text-align: center;
+    margin: 5rem;
+}
+
+.add-dev-btn {
+    display: block;
+    margin: 1rem auto 6rem auto;
+    padding: 1rem 2rem;
+    background-color:bisque;
+    
+    border: none;
+    border-radius: 1rem;
+    box-shadow: 2px 2px 4px #36454f;
+
+    font-size: 1.5rem;
+    font-weight: bold;
+}
+
+.add-dev-btn:hover,
+.add-dev-btn:focus {
+    cursor: pointer;
+    background-color: #36454f;
+    color: bisque;
+    transform: scale(1.3);
+
+}
+
+.modal {
+    /* display: none; */
+    width: 30%;
+    height: 100%;
+
+    background-color: blueviolet;
+    color: whitesmoke;
+
+    margin: 0 auto;
+    padding: 1rem;
+    border-radius: 1.25rem;
+}
+
+.form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    gap: 1rem;
+    width: 50%;
+    margin: 0 auto;
+}
+
+.input-form {
+    width: 100%;
+    padding: .5rem;
+    border: none;
+    border-radius: .5rem;
+    box-shadow: 2px 2px 4px #36454f;
+    font-size: 1rem;
+}
+
+label {
+    font-size: 1.5rem;
+}
+
```

Comment:
I'm not seeing where the styles below this line are applied... Were you testing out how contributors info would be displayed?